cmd/rollapp/init/init.go

```diff
@@ -126,6 +127,12 @@ func Cmd() *cobra.Command {
 					Show()
 			}
 
+			das := []string{"celestia", "avail"}
```

Comment:
da comes from rollappparams of the genesis file and is set via the portal, roller should respect whatever is set there during init

https://github.com/dymensionxyz/roller/blob/main/utils/genesis/genesis.go#L70-L76